node/pkg/driver/node_utils.go

```diff
@@ -196,10 +177,50 @@ func (n NodeUtils) StageInfoFileIsExist(filePath string) bool {
 	}
 	return true
 }
+func readFile(path string) (string, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return "", err
+	}
+
+	defer file.Close()
+
+	fileOut, err := ioutil.ReadAll(file)
+	if err != nil {
+		return "", err
+	}
+
+	fileOutString := string(fileOut)
+
+	return fileOutString, nil
+}
+
+func readAfterPrefix(path string, prefix string, portType string) (string, error) {
+	fileOutString, err := readFile(path)
+	if err != nil {
+		return "", err
+	}
+
+	if prefix != "" {
+		if !strings.Contains(fileOutString, prefix) {
+			return "", fmt.Errorf(ErrorWhileTryingToReadPort, portType, fileOutString)
+		}
+		fileOutString = strings.TrimPrefix(fileOutString, prefix)
+	}
+
+	trimmedOut := strings.TrimSpace(fileOutString)
```

Comment:
while you're at it, could we lose the "out" occurrences in this function?
I don't know why an input file name has "out", nor why a trimmed string is trimmed "out" (the space would be trimmed "off", but we don't need a name for the ignored space)

Reply:
input file name?

Reply:
sorry, input file variable name

Reply:
we are only reading, its output file..

Reply:
output file is where a program would write to

Reply:
its the output of the file.. 
anyway, what are you suggesting?

Reply:
file doesn't do anything, so it doesn't have an "output".
I suggest removing the "out" part in the names

Reply:
done